dotnetv3/Lambda/CreateDynamoDBTableExample/CreateDynamoDBTableExample/Function.cs

```diff
@@ -0,0 +1,103 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.RuntimeSupport;
+using Amazon.Lambda.Serialization.SystemTextJson;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+// This project specifies the serializer used to convert Lambda event into .NET classes in the project's main 
```

Comment:
a Lambda event

Reply:
main is repeated

Reply:
Fixed both errors. I also re-flowed the lines so that they aren't quite so wide.